src/helpers/ClauseResultsHelpers.ts

```diff
@@ -132,11 +132,16 @@ export function findAllLocalIdsInStatement(
         aliasMap[v] = statement.expression.localId;
         // Determine the localId for this alias.
         if (statement.localId) {
-          // Older translator versions require with statements to use the statement.expression.localId + 1 as the alias Id
-          // even if the statement already has a localId. There is not a clear mapping for alias with statements in the new
-          // translator, so they will go un highlighted but this will not affect coverage calculation
+          // There is not a clear mapping for `With` and `Without` relationship alias with statements in newer
+          // translator versions. The node in the annotation that contains the alias does has a local id that doesn't
+          // exist in the elm. We have to find this localId by looking for it in the annotation structure.
           if (statement.type === 'With' || statement.type === 'Without') {
-            alId = (parseInt(statement.expression.localId, 10) + 1).toString();
+            if (annotation) {
```

Comment:
Since we've added a nested `if` here, we should probably make sure we have a default value for `alId` when there isn't an annotation or when we otherwise can't find the relationship alias, otherwise `alId` will be undefined. On line 36 of this file, we have a check against null, but we don't seem to have a check against undefined.

Reply:
Recommended change: Update check on line 36 to also encompass the undefined case.